### PR TITLE
CVSL-441: Upgrade docker containers for ARM support

### PR DESCRIPTION
**Pull request by smcveigh941**
- Apple have stopped manufacturing Macbooks with intel x86_64 chips
- New Macbooks have ARM architecture with the new M1 chip
- Some of the docker containers we use locally have not been packaged for ARM architecture, which means that running them locally either fails or is very slow, as the x86_64 architecture needs to be emulated
- Newer versions of these docker images do support ARM, so upgrading to those new versions solves this issue

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -141,17 +141,17 @@ jobs:
             export AWS_ACCESS_KEY_ID=foo
             export AWS_SECRET_ACCESS_KEY=bar
 
-            aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name create_and_vary_a_licence_prison_events_queue
-            aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name create_and_vary_a_licence_prison_events_queue_dl
-            aws --endpoint-url=http://localhost:4576 sqs set-queue-attributes --queue-url "http://localhost:4576/queue/create_and_vary_a_licence_prison_events_queue" --attributes '{"RedrivePolicy":"{\"maxReceiveCount\":\"3\", \"deadLetterTargetArn\":\"arn:aws:sqs:eu-west-2:000000000000:create_and_vary_a_licence_prison_events_queue_dl\"}"}'
+            aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name create_and_vary_a_licence_prison_events_queue
```
Comment (smcveigh941) [inline .circleci/config.yml:144]: Port 4576 is deprecated in `v0.11.2` and removed in `v0.13.2`, use port 4566

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '3.1'
 services:
 
   redis:
-    image: 'bitnami/redis:5.0'
+    image: 'redis:5.0'
```
Comment (smcveigh941) [inline docker-compose-dev.yml:5]: The bitnami distribution does not build containers with ARM support. Recommended to use redis from the official source. This is only used for running locally anyway

File: helm_deploy/create-and-vary-a-licence/values.yaml [hunk 3]
```diff
@@ -96,7 +96,7 @@ gotenberg:
 
   image:
     repository: thecodingmachine/gotenberg
-    tag: 6.3.0
+    tag: 7.5.0
```
Comment (smcveigh941) [inline helm_deploy/create-and-vary-a-licence/values.yaml:99]: gotenberg is built with ARM support from `7.5.0` onwards

**Review by timharrison-moj — COMMENTED** [review]
LGTM.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**